backend/api/chat_orchestrator.py

```
@@ -450,14 +468,20 @@ async def generate_scene_introduction_enhanced(self) -> str:
                     except (json.JSONDecodeError, TypeError) as parse_err:
-                        print(f"Unable to parse previous summary JSON: {parse_err}")
+                        debug_log(f"Unable to parse previous summary JSON: {parse_err}")
             
             return intro
             
         except Exception as e:
-            print(f"Error generating enhanced scene introduction: {e}")
-            # Fallback to basic introduction
-            return self.generate_scene_introduction()
+            debug_log(f"Error generating enhanced scene introduction: {e}")
+            raise e
```

Ruff (0.14.1)

Use `raise` without specifying exception name

Remove exception name

(TRY201)